Modelplace-Cloud-API-Python/README.md

```diff
@@ -0,0 +1,69 @@
+# Modelplace cloud API
+
+This page contains a sample Python application using the Modelplace.AI cloud API. 
+
+Our API is a simple HTTP interface with an API key and an image or video as input and JSON with results and visualization as output. Explore our [documentation] and this example to integrate artificial intelligence models into your application or workflow.
```

Comment:
Modelplace.AI Cloud API is a simple HTTP interface with an API key and an image or video as input and JSON with results and visualization as output. Explore [documentation] and this Python example to integrate AI models into your application or workflow.